.nais/test/klass-forvaltning.yaml

```diff
@@ -14,6 +14,7 @@ spec:
     outbound:
       external:
         - host: auth.test.ssb.no
+        - application: klass-solr
```

Comment:
This shouldn't be in the `outbound.external` block, check the config in klass-api